src/xunit.performance.api/XunitPerformanceHarnessOptions.cs

```diff
@@ -104,6 +105,10 @@ public static XunitPerformanceHarnessOptions Parse(string[] args)
                                     Console.WriteLine(Usage());
                                     Environment.Exit(0);
                                     break;
+                                case ErrorType.VersionRequestedError:
```

Comment:
Can you tell me more about the expectation here?  When would this error be printed?  It looks like right now, the error is just a version number, but doesn't say what the error is.